---
source: apollo-federation/src/sources/connect/validation/mod.rs
expression: "format!(\"{:#?}\", errors)"
input_file: apollo-federation/src/sources/connect/validation/test_data/invalid_selection_arrow_method_arity.graphql
---
[
    Message {
        code: IncorrectArity,
        message: "`@connect(selection:)` on `Query.me` calls method `size` with the wrong number of arguments - expected 0, found 1",
        locations: [
            LineColumn {
                line: 11,
                column: 20,
            }..LineColumn {
                line: 14,
                column: 12,
            },
        ],
    },
    Message {
        code: UnresolvedField,
        message: "No connector resolves field `User.id`. It must have a `@connect` directive or appear in `@connect(selection:)`.",
        locations: [
            LineColumn {
                line: 19,
                column: 5,
            }..LineColumn {
                line: 19,
                column: 12,
            },
        ],
    },
    Message {
        code: UnresolvedField,
        message: "No connector resolves field `User.invalid`. It must have a `@connect` directive or appear in `@connect(selection:)`.",
        locations: [
            LineColumn {
                line: 20,
                column: 5,
            }..LineColumn {
                line: 20,
                column: 20,
            },
        ],
    },
]
